tests/playwright/pages/wp-admin-page.ts

```diff
@@ -42,6 +42,16 @@ export default class WpAdminPage extends BasePage {
 		return new EditorPage( this.page, this.testInfo );
 	}
 
+	async convertFromGutenberg() {
+		await this.page.click( '#elementor-switch-mode' );
+		await this.page.waitForLoadState( 'networkidle', { timeout: 20000 } );
```

Comment:
What is `waitForLoadState( 'networkidle'` for? I don't see an Ajax call when clicking on `Edit with Elementor` 🤔 But if there is one, maybe we can do:

await Promise.all( [
	page.waitForResponse( '/wp-admin/admin-ajax.php' ),
	....
] );

(Similar to `addDynamicWidget()`)

I'm just asking because this 20 second timeout feels long. WDYT?

Reply:
Update: Sorry I had a filter on my XHR requests 🙈 I see there is an Ajax call to `wp-json/wp/v2/pages/`. So actually, same question, but just with this endpoint.